gulpfile.js

```diff
@@ -31,6 +33,7 @@ var WATCH_INTERVAL    = 1000; // in milliseconds
 var ROOT_DIR   = ".";
 var SOURCE_DIR = path.join(ROOT_DIR, "www");
 var DEV_DIR    = path.join(ROOT_DIR, "build-dev");
+var LINT_DIR    = path.join(DEV_DIR, "blog");
```

Comment:
The directory where we want to run the lint. We can change it later as per use but currently the docs have a lot of formatting errors so I excluded it from linting.